WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -719,6 +719,7 @@ - (void)showBlogSelector
                                                                                        dismissHandler:dismissHandler];
     vc.title = NSLocalizedString(@"Select Site", @"");
     vc.displaysPrimaryBlogOnTop = YES;
+    vc.displaysCancelButton = [UIDevice isPhone];
```

Comment:
Apologies about that. Was the result of a direct translation, and didn't even realize we now have Multitasking.

Fixing, thank you!